bundleCleanup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+BUNDLE_LIFE_PERIOD="$1"
+
+if [ $BUNDLE_LIFE_PERIOD -gt 0 ]; then
+  echo "deleting bundles older than $BUNDLE_LIFE_PERIOD seconds"
+else
+  BUNDLE_LIFE_PERIOD=300
+  echo "deleting bundles older than $BUNDLE_LIFE_PERIOD seconds"
+fi
+
+CURRENT_TIME=$(date +%s)
+
+for DIR in prebid.js/prebid_*;
+  do
+    echo "checking $DIR"
+    BUNDLE_DIR="${DIR}/build/dist/*"
```

Comment:
This catches all the file under `build/dist`, and all files are being deleted - that's not exactly what we want to happen. Only files named like `prebid.<uuid>.js` in that directory should be subject for deletion.

Reply:
Done

Reply:
Done